Prefetching-Library/android_prefetching_lib/src/main/java/nl/vu/cs/s2group/nappa/room/converter/DateConverters.java

```diff
@@ -0,0 +1,18 @@
+package nl.vu.cs.s2group.nappa.room.converter;
+
+import androidx.room.TypeConverter;
+
+import java.util.Date;
+
+public class DateConverters {
```

Comment:
Provides automatic type conversion between Room (database) and the Application models